libmisc/xmalloc.c

```diff
@@ -66,3 +66,10 @@
 {
 	return strcpy (xmalloc (strlen (str) + 1), str);
 }
+
+void xfree(/*notnull@*/void *ap)
```

Comment:
But ...  in fact it *can* be null :)  Then we just do nothing.

Reply:
i think so :).